src/sass/calendar/_base.scss

```diff
@@ -0,0 +1,34 @@
+// Copyright (C) 2018 The Trustees of Indiana University
+// SPDX-License-Identifier: BSD-3-Clause
+
+@use 'core' as *;
+
+.#{$prefix}-cal {
+  font-size: $ts-32;
+  font-weight: $font-weight-bold;
+  background-color: $color-black-000;
+  display: inline-flex;
+  padding: .75rem $spacing-sm;
+  line-height: 1.1;
+  position: relative;
+  border-top: $spacing-sm solid $color-crimson-dark-base;
+  flex-direction: column;
+
+  &__month {
+    color: $color-yellow-700;
+    font-size: $ts-14;
+    text-transform: uppercase;
+    padding: 0 $spacing-sm;
+    margin-left: auto;
+    margin-right: auto;
+  }
+
+  &__day {
+    font-weight: $font-weight-regular;
+    font-size: $ts-16 * 1.5;
```

Comment:
I think you could get away with using a standard value from the type scale here instead of introducing a magic number. There are definitely times when that needs to happen, but in this case the visual difference between the result of this calculation (24px/1.5rem) and the standard `$ts-23` value (23px/1.4375rem) is so close that the trade-off might not be worth it.